functions_and_scope_assingnment.js

```diff
@@ -0,0 +1,329 @@
+// // # Function Exercises
+// //
+// // ## Question 1
+// //
+// // Write a function that returns the largest of two numbers. If they are equal, return the second one.
+// //
+// // Example:
+// //
+// // Input: `12, 4`
+// //
+// // Output: `12`
+function biggerAndSmaller(num1, num2){
```

Comment:
Small thing but when the question asks for return value, return a value out of function.